README.md

```diff
@@ -99,6 +111,20 @@ signed by the server certificate.
 
 It requires a server certificate with optional certificate authority certificate for the web server and a certificate authority certificate for the validation of client certificates if the client authorization is not disabled.
 
+### HSM Signing
+
+Using the `hsm` signing options signing is switched to hardware-based signing,
+no private key is required anymore. The signing algorithms are the same.
+
+With `--hsm-module` the path to the HSM shared library is specified. This
+library is specific for your hardware signing module, which should be used for
+signing. For testing, you can use the `softhcm` library (for Unix systems
+this is typically `/usr/lib/softhsm/libsofthsm2.so`)
```

Comment:
Shouldn't this section be located under "Signing Tool" before "Server"?

Reply:
Signing Tool and signing server are two flavors for using the executable, So, then it should be moved in front of the tool section.